sevntu-checks/src/test/java/com/github/sevntu/checkstyle/checks/coding/ForbidCertainMethodsCheckTest.java

```diff
@@ -0,0 +1,286 @@
+            "cannot initialize module com.puppycrawl.tools.checkstyle.TreeWalker - "
+            + "Cannot set property 'argumentCount' to 'badStart-badEnd' in module "
+            + "com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck");
+    }
+
+    @Test
+    public void testBadRange3ForArgumentCount() throws Exception {
+        testBadRangeInner("2, badStart-badEnd",
+            "cannot initialize module com.puppycrawl.tools.checkstyle.TreeWalker - "
+            + "Cannot set property 'argumentCount' to '2, badStart-badEnd' in module "
+            + "com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck");
+    }
+
+    @Test
+    public void testBadRange4ForArgumentCount() throws Exception {
+        testBadRangeInner(" , 1-4 ",
+            "cannot initialize module com.puppycrawl.tools.checkstyle.TreeWalker - "
+            + "Cannot set property 'argumentCount' to ' , 1-4 ' in module "
+            + "com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck");
+    }
+
+    @Test
+    public void testUnboundedRangeForArgumentCount() throws Exception {
+        testBadRangeInner(" - ",
+            "cannot initialize module com.puppycrawl.tools.checkstyle.TreeWalker - "
+            + "Cannot set property 'argumentCount' to ' - ' in module "
+            + "com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck");
+    }
+
+    @Test
+    public void testRangeWithBadBoundsForArgumentCount() throws Exception {
+        testBadRangeInner("10-1",
+            "cannot initialize module com.puppycrawl.tools.checkstyle.TreeWalker - "
+            + "Cannot set property 'argumentCount' to '10-1' in module "
+            + "com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck");
+    }
+
+    private void testBadRangeInner(String value, String expectedMessage) throws Exception {
+        final DefaultConfiguration checkConfig =
+            createModuleConfig(ForbidCertainMethodsCheck.class);
+        checkConfig.addAttribute("methodName", "exit");
+        checkConfig.addAttribute("argumentCount", value);
+        final String[] expected = {
+        };
+        try {
+            verify(checkConfig, getPath("InputForbidCertainMethodsCheck.java"), expected);
+            Assert.fail("Expecting CheckstyleException");
+        }
+        catch (CheckstyleException ex) {
+            checkExceptionMessage(ex,
+                expectedMessage);
+        }
+    }
+
+    @Test
+    public void testRangeForArgumentCount() throws Exception {
+        testArgumentCountInner("0-1");
+    }
+
+    @Test
+    public void testRangeWithSpaceForArgumentCount() throws Exception {
+        testArgumentCountInner("  0   - 1 ");
+    }
+
+    @Test
+    public void testLeftOpenRangeForArgumentCount() throws Exception {
+        testArgumentCountInner("-1");
+    }
+
+    private void testArgumentCountInner(String argumentCountRange) throws Exception {
```

Comment:
This can't be a test method as it is doing a `verify` with no exceptions. If we expand regression util to sevntu, we will never pick up this information.
This method must be inlined where it is used.
I am fine with `testBadRangeInner` for now since it is an exception case.

@romani I may need to expand the check to flag these type of methods.

Reply:
Done.